docs/docs/documentation.md

```
@@ -27,13 +27,21 @@ branch, to be served using [GitHub pages](https://docs.github.com/en/pages).
-2. Install [MkDocs](https://www.mkdocs.org/getting-started/) and plugins
+2. Install [MkDocs](https://www.mkdocs.org/getting-started/) and plugins via `pip` or `uv` package manager
 
     ```sh
-    pip install mkdocs mkdocs-material mkdocs-glightbox
+    uv add mkdocs mkdocs-material mkdocs-glightbox pre-commit
     ```
 
-3. Build and serve the docs at [http://127.0.0.1:8000/](http://127.0.0.1:8000/)
+3. Setup pre-commit hook
+
+	```sh
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)